docs/plugin_list.md

```diff
@@ -165,6 +165,12 @@ Clone repos from github & create GIT repos & use standard git commands. Also exp
 <br>[Profile name](https://github.com/ethereum/remix-plugins-directory/blob/master/plugins/wallet-connect/profile.json): walletconnect
 <br>[Make an issue](https://github.com/yann300/remix-walletconnect/issues)
 
+**Zksync** &nbsp; ![](images/pi-zksync.png) &nbsp;
+<br>Compile and deploy smart contracts for zkSync Era.
+<br>[Profile name](https://github.com/ethereum/remix-plugins-directory/blob/master/plugins/zksync/profile.json): ZkSync
```

Comment:
Thanks for this PR.  I just need you to make a small change - the profile name is zkSync not ZkSync.  Could you update this?  Thanks.  Please check the .json file to see the name.  Thanks again!!

Reply:
Done, please check again. @ryestew